src/test/regress/output/multi_subquery.source

```diff
@@ -590,20 +977,15 @@ FROM
 		events.composite_id <= '(1, 9223372036854775807)'::user_composite_type AND
 		event_type = 'pay') AS subquery_2
 	ON
-		subquery_1.tenant_id = subquery_2.tenant_id AND
-		subquery_1.user_id = subquery_2.user_id
+		subquery_1.composite_id = subquery_2.composite_id
 	GROUP BY
 		subquery_1.tenant_id,
 		subquery_1.user_id,
 		hasdone) AS subquery_top
 GROUP BY
 	hasdone;
-   event_average    |       hasdone       
---------------------+---------------------
- 4.0000000000000000 | Has not done paying
- 2.5000000000000000 | Has done paying
-(2 rows)
-
+ERROR:  cannot pushdown the subquery since all relations are not joined using distribution keys
```

Comment:
It is not possible to forget that since we've already open PR which enables those tests: #1289 
I think to add notes only could lead to git conflicts so I'm refusing to handle this request 😄